src/main/proto/control.proto

```diff
@@ -33,6 +33,9 @@ message PlatformInboundInstruction {
            This information is used by AxonServer to monitor the progress of Tracking Processors across instances.
          */
         EventProcessorInfo event_processor_info = 2;
+
+        /* This heartbeat is used by AxonServer in order to check the connection is still alive*/
```

Comment:
Nit: I'd rephrase slightly by adding `... in order to check *if* the connection is...`.